nimare/dataset/dataset.py

```diff
@@ -88,3 +222,47 @@ def get_images(self):
 
     def get_coordinates(self):
         pass
+
+    def save(self, filename):
```

Comment:
Addressing concerns raised in #2, I replaced the filename-based check with a `compress` argument in my new PR (#5).